functions/config.bash

```diff
@@ -33,7 +33,14 @@ clean_config_userpw() {
 ## Update java architecture in config file
 ## Valid options: "32-bit" & "64-bit"
 update_config_java() {
-  cond_redirect grep -q '^java_arch' "$CONFIGFILE" && sed -i "s/^java_arch.*/java_arch=$1/" "$CONFIGFILE" || echo "java_arch=$1" >> "$CONFIGFILE"
+  if [ "$1" == "64-bit" ] && is_pithreeplus || is_pifour || is_armv7l || is_armv6l; then
```

Comment:
Is the logic correct?
The test build failed because your code installed 64bit as requested:
https://travis-ci.com/github/openhab/openhabian/jobs/326715364#L1229
But it should have recognized that und downgraded to 32bit.

FWIW use mode=debug_maximum in openhabian.conf if you want debug output of every command in the test build to find the mistake.

Reply:
Ok, so I'm debugging the implementation right now and will let you know when it works or if I need some help.

Reply:
Latest version still fails to correct 64->32bit on arm32hf
See this (from auto build upon your last upload)
https://travis-ci.com/github/openhab/openhabian/jobs/326831221#L1522